tsconfig.json

```diff
@@ -1,22 +1,30 @@
 {
 	"compilerOptions": {
-		"allowJs": true,
-		"allowSyntheticDefaultImports": true,
-		"esModuleInterop": true,
-		"forceConsistentCasingInFileNames": true,
+		"target": "ES2022",
+		"useDefineForClassFields": true,
+		"lib": ["ES2022", "DOM", "DOM.Iterable"],
+		"module": "ESNext",
+		"skipLibCheck": true,
+
+		/* Bundler mode */
+		"moduleResolution": "bundler",
+		"allowImportingTsExtensions": true,
```

Comment:
This is a layover from the default vite typescript config file when bootstrapping a vite react typescript app. When @slifty and I first went through this process we wanted to follow their setup as closely as possible, but since we know we want ESLint specifically to disallow this I think it's fine to remove it!